src/CoreShop/Bundle/CoreBundle/Migrations/Version20180615093141.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace CoreShop\Bundle\CoreBundle\Migrations;
+
+use CoreShop\Component\Pimcore\DataObject\ClassUpdate;
+use Doctrine\DBAL\Schema\Schema;
+use Pimcore\Migrations\Migration\AbstractPimcoreMigration;
+use Symfony\Component\DependencyInjection\ContainerAwareInterface;
+use Symfony\Component\DependencyInjection\ContainerAwareTrait;
+
+class Version20180615093141 extends AbstractPimcoreMigration implements ContainerAwareInterface
+{
+    use ContainerAwareTrait;
+
+    private const CORE_SHOP_PRODUCT_FIELDS = [
```

Comment:
`private const`  is not supported in PHP 7.0, anyway, a const is not even necessary here, can't you just create the variable inline and use it then?